src/components/workflows-configuration.js

```diff
@@ -0,0 +1,584 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import DeleteIcon from '@material-ui/icons/Delete';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import { FormattedMessage } from 'react-intl';
+import { addProcess, deleteProcess, fetchMergeConfigs } from '../utils/api';
+import { initProcesses } from '../redux/actions';
+import { useDispatch, useSelector } from 'react-redux';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Radio from '@material-ui/core/Radio';
+import { v4 as uuidv4 } from 'uuid';
+
+const useStyles = makeStyles(() => ({
+    addNewTso: {
+        border: '1px solid #ccc',
+        padding: '5px 0 20px 15px',
+        margin: '15px 0',
+    },
+    newTsoContainerLabel: {
+        fontWeight: 'bold',
+        padding: '10px 10px 0px 5px',
+    },
+    newTsoLabel: {
+        paddingLeft: '10px',
+    },
+    input: {
+        '&::placeholder': {
+            fontSize: '12px',
+        },
+    },
+    addNewArea: {
+        margin: '10px 0px 20px 0px',
+        width: '100%',
+    },
+}));
+
+const styles = (theme) => ({
+    root: {
+        margin: 0,
+        padding: theme.spacing(2),
+    },
+    closeButton: {
+        position: 'absolute',
+        right: theme.spacing(1),
+        top: theme.spacing(1),
+        color: theme.palette.grey[500],
+    },
+});
+
+const CustomDialogTitle = withStyles(styles)((props) => {
+    const { children, classes, onClose, ...other } = props;
+    return (
+        <DialogTitle
+            disableTypography
+            className={classes.root}
+            {...other}
+            style={{ padding: '15px' }}
+        >
+            <Typography variant="h6">{children}</Typography>
+            {onClose ? (
+                <IconButton
+                    aria-label="close"
+                    className={classes.closeButton}
+                    onClick={onClose}
+                >
+                    <CloseIcon />
+                </IconButton>
+            ) : null}
+        </DialogTitle>
+    );
+});
+
+const CustomDialog = withStyles(() => ({
+    paperScrollPaper: {
+        minWidth: '850px',
```

Comment:
minWidth 850px means this can never be nice on small screens...

Reply:
@med-hedi

Reply:
fixed